app/assets/javascripts/night_mode.js

```diff
@@ -0,0 +1,37 @@
+function set_night_mode(){
+	Cookies.set("night_mode",true)
+	document.documentElement.setAttribute('data-theme', 'dark')
+	$("#fs").prop("checked",true)
+
+
+}
+
+function set_day_mode(){
+	Cookies.set("night_mode",false)
+	document.documentElement.setAttribute('data-theme', 'light')
+	$("#fs").prop("checked",false)
+}
+
+function set_mode(night_mode){
+	if(night_mode){
+		console.log("goodnight")
+		set_night_mode();
+				}
+	else{
+		console.log("good morning")
+		set_day_mode();
+		}
+
+
+}
```

Comment:
```
function set_mode(night_mode)
{
        var mode;
	if(night_mode)
        {
		console.log("goodnight");
                mode = 'dark';
	}
	else
        {
		console.log("good morning");
                mode = 'light';
	}
        Cookies.set("night_mode",night_mode)
        document.documentElement.setAttribute('data-theme', mode)
        $("#fs").prop("checked",night_mode)
}
```

Reply:
Io lo modificherei così in modo da togliere i due metodi `set_night_mode()` e `set_day_mode()` e ridurre le azioni duplicate.

Reply:
umh, inizialmente quelle funzioni erano diverse tra loro quindi le avevo separate. adesso non ha molto più senso.